### PR TITLE
[4.0] Batch update urls

### PR DESCRIPTION
**Pull request by brianteeman**
Update the code so that it actually works 

Pull request for #24861 

Make layout match similar forms

Improve accessibility by using buttons for buttons

As far as I can tell it should be called "Batch Update Selected URL(s)" not "new URL(s)" - Based on the key being COM_REDIRECT_BATCH_UPDATE_WITH_NEW_URL I think it was supposed to be "batch update WITH new URL". I believe the proposed change is more accurate as you have to select redirects before they can be updated

### CODE REVIEW

File: administrator/components/com_redirect/tmpl/links/default_addform.php [hunk 1]
```diff
@@ -12,31 +12,44 @@
...
-					<div class="control-group">
-						<label for="comment" class="control-label"><?php echo Text::_('COM_REDIRECT_FIELD_COMMENT_LABEL'); ?></label>
-						<div class="controls">
-							<input type="text" name="comment" id="comment" value="" size="50" title="<?php echo Text::_('COM_REDIRECT_FIELD_COMMENT_DESC'); ?>">
-						</div>
-					</div>
-					<button class="btn btn-primary" type="button" onclick="this.form.task.value='links.duplicateUrls';this.form.submit();"><?php echo Text::_('COM_REDIRECT_BUTTON_UPDATE_LINKS'); ?></button>
-				</fieldset>
+	<div class="accordion-heading">
+		<button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#batch" aria-expanded="false" aria-controls="collapseExample">
```
Comment (Quy) [inline administrator/components/com_redirect/tmpl/links/default_addform.php]:
```suggestion
		<button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#batch" aria-expanded="false" aria-controls="batch">
```

File: administrator/components/com_redirect/tmpl/links/default_addform.php [hunk 2]
```diff
@@ -12,31 +12,44 @@
...
+				<div class="control-label">
+					<label id="new_url-lbl" for="new_url">
+						<?php echo Text::_('COM_REDIRECT_FIELD_NEW_URL_LABEL'); ?>
+					</label>
+				</div>
+				<div class="controls">
+					<input type="text" name="new_url" id="new_url" value="" size="50">
+				</div>
+				<div id="new_url-desc">
+					<small class="form-text muted">
```
Comment (Quy) [inline administrator/components/com_redirect/tmpl/links/default_addform.php]:
```suggestion
					<small class="form-text text-muted">
```

Comment (brianteeman) [reply administrator/components/com_redirect/tmpl/links/default_addform.php]: Pretty sure my code is correct

Comment (Quy) [reply administrator/components/com_redirect/tmpl/links/default_addform.php]: There is a `.text-muted` but not `.muted`.

```
.text-muted {
  color: #6f7780 !important; }
```

Comment (brianteeman) [reply administrator/components/com_redirect/tmpl/links/default_addform.php]: Sorry.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c4d6c36c9b0d4309d64eb4f8753da5e6b7a7b530<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25247">issues.joomla.org/tracker/joomla-cms/25247</a>.</sub>

**Comment by travisrisner** [comment]
I have tested this item :white_check_mark: successfully on c4d6c36

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25247">issues.joomla.org/tracker/joomla-cms/25247</a>.</sub>

**Comment by roland-d** [comment]
@brianteeman When testing this I do not get a modal but the form opens up below the button:
image

This is not how it is supposed to work right? I do not have any JavaScript errors.

**Comment by Quy** [comment]
It is an accordion and behaves just like in J3.

**Comment by brianteeman** [comment]
It is the intended behaviour (dont' know why it was created like that)

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**